java/README.md

```diff
@@ -1,14 +1,42 @@
-Prerequisites
--------------
+# CuVS Java API
 
-* JDK 22
-* Maven 3.9.6 or later
+CuVS Java API attempts to provide a Java based simple, efficient, and a robust vector search API.
 
-To build this API, please do `./build.sh java` in the top level directory. Since this API is dependent on `libcuvs` it must be noted that `libcuvs` gets built automatically before building this API.
+> [!CAUTION]
+> This is an experimental release and updates to this API should be expected in the coming days.
 
-Alternatively, please build libcuvs (`./build.sh libcuvs` from top level directory) before building the Java API with `./build.sh` from this directory.
 
-Building
---------
+## Prerequisites
 
-`./build.sh` will generate the `libcuvs_java.so` file in the `internal/` directory, and then build the final jar file for the cuVS Java API in the `cuvs-java/` directory.
+- gcc 11.4
+- [nvcc 12.4 or above](https://developer.nvidia.com/cuda-downloads)
```

Comment:
cuVS supports 11.x and box. Is there a reason the java API only supports 12.4? If not, let's just point to the cuVS `build from source` docs here and not duplicate these things.